Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -117,13 +119,17 @@ protected bool TestObjectExists(StorageService service, string bucket, string ob
     /// <example>
     ///   <para>Upload a local log file to GCS.</para>
     ///   <para><code>PS C:\> New-GcsObject -Bucket "widget-co-logs" -ObjectName "log-000.txt" `</code></para>
-    ///   <para><code>    -File "C:\logs\log-000.txt"</code></para>
+    ///   <para><code>    -Path "C:\logs\log-000.txt"</code></para>
     /// </example>
     /// <example>
     ///   <para>Pipe a string to a a file on GCS. Sets a custom metadata value.</para>
     ///   <para><code>PS C:\> "Hello, World!" | New-GcsObject -Bucket "widget-co-logs" -ObjectName "log-000.txt" `</code></para>
     ///   <para><code>    -Metadata @{ "logsource" = $env:computername }</code></para>
     /// </example>
+    /// <example>
+    ///   <para>Upload a folder to GCS</para>
+    ///   <para><code>PS C:\> New-GcsObject -Bucket "widget-co-logs" -Folder "C:\Folder"</code></para>
```

Comment:
`C:\Folder` doesn't quite fit with the example. How about a PowerShell equiv of: `%SystemDrive%\inetpub\logs\LogFiles`. (Where IIS logs go.)